### PR TITLE
feat: added Wishlist management

### PR DESCRIPTION
**Pull request by xii-vi**
### Remove from wishlist
ezgif com-gif-maker (6)

### CODE REVIEW

File: src/apis/apis.js [hunk 1]
```diff
@@ -76,10 +76,36 @@ const DecreaseCartQuantity = async (productData, encodedToken) => {
...
+         dataToUpdate.wishlist = [...res.data.wishlist]
+         localStorage.setItem('userData', JSON.stringify(dataToUpdate))
+         console.log(dataToUpdate);
+    }
+    catch {
+        console.log(error);
+    }
+
+}
+const RemoveFromWishlist = async (data, encodedToken) => {
```
Comment (TejasShekar) [inline src/apis/apis.js]: The function is empty. Fix it or remove it if its a redundant code

File: src/pages/wishlist/wishlist.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { useCart } from "../../context/cartContext";
+import { WishlistItem } from "../../component/card/wishlistCard";
+
+export const Wishlist = () => {
+const {
+wishlistState: { wishlistItem },
+} = useCart();
+console.log(wishlistItem);
```
Comment (TejasShekar) [inline src/pages/wishlist/wishlist.jsx]: Remove console logs

**Review by TejasShekar — APPROVED** [review]
Have left a few important suggestions. Go through them.

**Review by haarsh24 — APPROVED** [review]
1. UI Looks amazing
2. clean code good file folder structure
3. everything looks good

**Pull request merged**